test/unit/core/bit_manipulation_test.cpp

```diff
@@ -27,47 +31,46 @@ TEST(bit_manipulation, sizeof_bits)
 
 TEST(bit_manipulation, is_power_of_two)
 {
-    constexpr bool is_power_of_two0 = seqan3::detail::is_power_of_two(0);
-    constexpr bool is_power_of_two1 = seqan3::detail::is_power_of_two(1);
-    constexpr bool is_power_of_two2 = seqan3::detail::is_power_of_two(2);
-    constexpr bool is_power_of_two3 = seqan3::detail::is_power_of_two(3);
+    constexpr bool is_power_of_two0 = std::has_single_bit(0u);
+    constexpr bool is_power_of_two1 = std::has_single_bit(1u);
+    constexpr bool is_power_of_two2 = std::has_single_bit(2u);
+    constexpr bool is_power_of_two3 = std::has_single_bit(3u);
     EXPECT_FALSE(is_power_of_two0);
     EXPECT_TRUE(is_power_of_two1);
     EXPECT_TRUE(is_power_of_two2);
     EXPECT_FALSE(is_power_of_two3);
 
     for (size_t power_of_two = 1; power_of_two <= (size_t{1u} << 31); power_of_two <<= 1)
     {
-        EXPECT_TRUE(seqan3::detail::is_power_of_two(power_of_two));
+        EXPECT_TRUE(std::has_single_bit(power_of_two));
 
         size_t next_power = (power_of_two << 1);
         for (size_t i = power_of_two + 1, k = 0; i < next_power && k < max_iterations; ++i, ++k)
         {
-            EXPECT_FALSE(seqan3::detail::is_power_of_two(i)) << i << " should not be a power of two.";
+            EXPECT_FALSE(std::has_single_bit(i)) << i << " should not be a power of two.";
         }
     }
 }
 
 TEST(bit_manipulation, next_power_of_two)
 {
-    constexpr size_t next_power_of_two0 = seqan3::detail::next_power_of_two(0);
-    constexpr size_t next_power_of_two1 = seqan3::detail::next_power_of_two(1);
-    constexpr size_t next_power_of_two2 = seqan3::detail::next_power_of_two(2);
-    constexpr size_t next_power_of_two3 = seqan3::detail::next_power_of_two(3);
+    constexpr size_t next_power_of_two0 = std::bit_ceil(0u);
```

Comment:
shouldn't the test also be moved to the std folder?

Reply:
They would be deleted when we remove the deprecated functions